tmt/utils.py

```diff
@@ -426,9 +381,15 @@ class Common:
     # mostly fine.
     parent: Optional['Common'] = None
 
+    def inject_logger(self, logger: Optional[tmt.log.Logger] = None) -> None:
```

Comment:
Please, add a short docstring.

Reply:
Why is the `logger` parameter optional?

Reply:
Relict of the distant past. Note the `assert` on the next line - I wanted to keep mypy happy, but I also wanted to catch code trying to sneak `None` as a logger. It should not be `None` anymore.

Reply:
Fixed in 39891de6dbf9fb4e01471bda84147df2156d9eea